llvm/test/Analysis/LoopAccessAnalysis/attr-mem-write-only.ll

```diff
@@ -0,0 +1,117 @@
+; RUN: opt < %s -mattr=+sve -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize -debug-only=loop-accesses -disable-output 2>&1 | FileCheck %s
```

Comment:
I think you should only be calling here : "-passes='print<access-info>' -debug-only=loop-accesses -disable-output"

Reply:
Indeed, using `-passes='print<access-info>'` is a better option here:
- it allows running LAA on the code regardless of LV's Legality result
- this would allow checking for `frexp` methods in a target agnostic manner.

Great suggestion, thanks!